examples/serverless-fastapi-python/serverless.yml

```diff
@@ -0,0 +1,84 @@
+service: ${env:SERVICE_NAME, 'serverless-fastapi-example'}
+
+frameworkVersion: '3'
+
+plugins:
+  - serverless-python-requirements
+  - serverless-offline
+  - ./plugins/serverless-offline-python-requirements
+  - serverless-dotenv-plugin
```

Comment:
we can avoid the `serverless-dotenv-plugin` because the new versions of serverless already handle .env files based on the stage when using `useDotenv: true` 👌🏻